src/homeContent.js

```diff
@@ -0,0 +1,25 @@
+const title = document.createElement('h1');
+const image = document.createElement('img');
+const para = document.createElement('p');
+
+const insertHome = (contentDiv, container) => {
+  // Title
+  title.innerHTML = 'Restaurant Project';
```

Comment:
You might prefer [`textContent`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Differences_from_innerHTML) over `innerHTML`.